docs/feature-flags/sdks/node.md

```diff
@@ -101,21 +101,32 @@ const variation = eppoClient.getAssignment(
 );
 ```
 
-The `getAssignment` function takes two required and one optional input to assign a variation:
+The `getStringAssignment` function takes two required and one optional input to assign a variation:
 
 - `subjectKey` - The entity ID that is being experimented on, typically represented by a uuid.
 - `flagOrExperimentKey` - This key is available on the detail page for both flags and experiments.
-- `targetingAttributes` - An optional map of metadata about the subject used for targeting. If you create rules based on attributes on a flag/experiment, those attributes should be passed in on every assignment call.
+- `subjectAttributes` - An optional map of metadata about the subject used for targeting. If you create rules based on attributes on a flag/experiment, those attributes should be passed in on every assignment call.
+
+### Typed assignments
+
+Additional functions are available:
+
+```
+getBoolAssignment(...)
+getNumericAssignment(...)
+getJSONStringAssignment(...)
+getParsedJSONAssignment(...)
+```
 
 ### Handling `null`
 
 We recommend always handling the `null` case in your code. Here are some examples illustrating when the SDK returns `null`:
 
 1. The **Traffic Exposure** setting on experiments/allocations determines the percentage of subjects the SDK will assign to that experiment/allocation. For example, if Traffic Exposure is 25%, the SDK will assign a variation for 25% of subjects and `null` for the remaining 75% (unless the subject is part of an allow list).
 
-2. If you are using Eppo for experiment assignments, you must start the experiment in the user interface before `getAssignment` returns variations. It will return `null` if the experiment is not running, both before and after.
+2. If you are using Eppo for experiment assignments, you must start the experiment in the user interface before `getStringAssignment` returns variations. It will return `null` if the experiment is not running, both before and after.
```

Comment:
This paragraph doesn't make sense to me. You can get assignments as long as the feature flag environment is enabled. You would continue to get assignments even if an experiment is not in a running state.

Am I reading this wrong?

Reply:
I'm confused too; I think this may refer to the "Eppo randomization" world prior to building out feature flags.

Reply:
I recall seeing something that the user can roll out the winning variation. If so, the experiment would still be "enabled" but "ended"? Is this true and if so should we update the docs to reflect this?

Reply:
I changed it to a sentence about making sure the environment is enabled to avoid getting null assignments. That much makes sense to me